src/mono/mono.proj

```diff
@@ -856,6 +856,8 @@
       <Output TaskParameter="ExitCode" PropertyName="MonoNinjaFound"/>
     </Exec>
     <PropertyGroup>
+      <_MonoVerboseArg Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(MonoNinjaFound)' == '0'">-v</_MonoVerboseArg>
+      <_MonoVerboseArg Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(MonoNinjaFound)' != '0'">VERBOSE=1</_MonoVerboseArg>
```

Comment:
I don't think we should turn on verbose build in CI by default, it just bloats the build log and makes it harder to understand what's going on.

I'm fine with having the option of course so you can enable it when something goes wrong :)